ACHNBrowserUI/ACHNBrowserUI/SceneDelegate.swift

```diff
@@ -14,9 +14,12 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
     
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
```

Comment:
try avoiding a forced cast
```swift
guard let context = (UIApplication.shared.delegate as? AppDelegate)?.persistentContainer.viewContext else {
    return // or fatalError("With an explicit message like: Cannot retrieve persistent container view context from UIApplication shared delegate")
}
```

Reply:
meh, if it fails then its because the app isn't working at all. :D

Reply:
what's an app without an app delegate? :P

Reply:
I'll agree on that. Sometimes it's better to have a crash and know about it that to have the app running in an unknown state. :p

Reply:
Agreed, that's why I purposed `fatalError("With an explicit message like: Cannot retrieve persistent container view context from UIApplication shared delegate")` instead of the `return` in second choice.
But you're right, it won't happen at any point, so maybe I'm too "against" force unwrap/force cast, etc... :D